src/FSharp.Data.GraphQL.Client/GraphQLClientConnection.fs

```diff
@@ -0,0 +1,19 @@
+/// The MIT License (MIT)
+/// Copyright (c) 2016 Bazinga Technologies Inc
+
+namespace FSharp.Data.GraphQL
+
+open System
+open System.Net.Http
+
+/// The connection component for GraphQL client operations.
+[<AllowNullLiteral>]
+type GraphQLClientConnection(client : HttpClient, ownsClient : bool) =
```

Comment:
Will it work making it `struct`?

Reply:
Well, we can. Will it be better as a `struct`?

Reply:
Yes, it is a single field. For performance struct is better

Reply:
The problem with making it a `struct` is that it does have a nullable value inside, and with a forced default constructor that does not initialize it, we can have a `GraphQLClientConnection` that actually holds no `HttpMessageInvoker`. I kinda don't know how to solve this problem. Any suggestion?